meetups/2023-11-16/announcement.md

```diff
@@ -0,0 +1,22 @@
+# Nix(OS) Learning Group Meetup #2
+
+- **Wann?** 16.11.2023, ca. 18:00-22:00
+- **Wo?** RUB, Raum MC 1.30 / MC 1.31. Eine Anfahrtsbeschreibung findet ihr auf der [Webseite der Uni](https://www.ruhr-uni-bochum.de/de/anreise-zur-ruhr-universitaet). Die Räume MC 1.30 / MC 1.31 finden sich sehr zentral im ersten Stock. Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.
+- **Wer?** Absolut jede*r mit Interesse an Nix(OS). Ihr müsst keine Vorerfahrung mit Nix mitbringen.
+
+### Was ist Nix?
+
+Nix ist ein Package Manager, mit dem du Software deklarativ, hermetisch und reproduzierbar bauen kannst. Die Software-Pakete werden dafür in der funktionalen Programmiersprache Nix beschrieben. NixOS ist eine Linux-Distribution, die auf Nix (Paket-Manager und Programmiersprache) aufbaut. Das ganze System wird deklarativ in einer Config beschrieben und lässt sich u.A. einfach reproduzieren, in Git verwalten, und Versionen können einfach zurückgerollt werden. Dabei bricht Nix mit etablierten Linux-Konzepten und eignet sich ideal, um sein Verständnis für Linux und Software generell zu vertiefen.
+Mehr Informationen zu Nix/NixOS gibt es u.A. auf der [offiziellen Website](https://nixos.org/).
+
+### Was ist die Learning Group?
+
+In regelmäßigen Treffen wollen wir mit euch zusammen unser Wissen über Nix(OS) vertiefen, zusammen an Systemkonfigurationen hacken und Software packagen. Explizit wollen wir auch Menschen ansprechen, die Nix noch nicht kennen, aber gerne ausprobieren würden und euch beim Einstig unterstützen.
+
+### Was solltest du mitbringen?
+
+Bring am besten einen Laptop mit Linux oder MacOS mit. Alternativ könnt ihr auch einen bootbaren USB-Stick mitbringen, oder euch eine VM aufsetzen. Wir helfen euch natürlich auch gerne, Nix/NixOS auf einem System, das ihr mitbringt, zu installieren.
```

Comment:
I'm not sure about adding macos here. Will you do the support for those people? Because I tried last time and I couldn't help.

Reply:
Yes. I can introduce Nix-Darwin